src/main/kotlin/no/nav/familie/ef/mottak/repository/domain/EttersendingDb.kt

```diff
@@ -0,0 +1,29 @@
+package no.nav.familie.ef.mottak.repository.domain
+
+import no.nav.familie.ef.mottak.encryption.FileCryptoConverter
+import no.nav.familie.ef.mottak.encryption.StringValCryptoConverter
+import java.time.LocalDateTime
+import java.util.*
+import javax.persistence.*
+
+@Entity
+@Table(name="ettersending")
+data class EttersendingDb (@Id
```

Comment:
Tenker denne kan hete `Ettersending`

Reply:
Da blir det to typer som heter ettersending. Ettersending til database og ettersending som dto. Har gjort slik nå at denne klassen heter ettersending også et ettersending dto objektet importert som ettersendingDto